main.go

```diff
@@ -38,12 +41,15 @@ var BootstrapAddresses = []string{
 var Info = log.New(ioutil.Discard, "I: ", 0)
 var Error = log.New(os.Stderr, "E: ", 0)
 
+const bufSize = 1 << 20
+
 func main() {
 	var flags flag.FlagSet
 	flags.Usage = func() {
-		fmt.Fprintln(flags.Output(), "Usage: p2pcat [-v] [-routed] MULTIADDR PROTOCOL")
+		fmt.Fprintln(flags.Output(), "Usage: p2pcat [-v] [-routed] -listen/MULTIADDR PROTOCOL")
```

Comment:
Nit:
```suggestion
		fmt.Fprintln(flags.Output(), "Usage: p2pcat [-v] [-routed] MULTIADDR PROTOCOL")
		fmt.Fprintln(flags.Output(), "  or:  p2pcat [-v] [-routed] -listen PROTOCOL")
```

(at least that's what some GNU commands do in cases like this)